File: Data Science and Machine Learning Essentials/Lab2D Data Cleansing and Transformation in Azure ML/DAT203x_Lab02D/CleanForestFires.py
def azureml_main(frame1):
  import pandas as pd
  import os.path

## Filter out outliers 
  frame1 = frame1[(frame1["FFMC"] > 40.0) & \
                                (frame1["ISI"] < 30.0) & \
                                (frame1["rain"] < 3.0)]
  return frame1

File: Data Science and Machine Learning Essentials/Lab2D Data Cleansing and Transformation in Azure ML/DAT203x_Lab02D/CleanForestFires.R
frame1 <- maml.mapInputPort(1)

## Remove outliers 
library(dplyr)
frame1 <- frame1 %>% filter(FFMC > 40) %>% 
                     filter(ISI < 30) %>% 
                     filter(rain < 3)

## Output the data frame
maml.mapOutputPort('frame1')
